File: Task-03/Python/subtask3.py
n = int(input("Enter the value n :"))
if(n%2!=0):
    for i in range(int((n+1)/2)):
        print(' '*(n-i-1), end=' ')
        print('*'*(2*i+1))
    
    for i in range(int((n-3)/2),-1,-1):
        print(' '*(n-i-1), end=' ')
        print('*'*(2*i+1))
else:
    for i in range(int((n+1)/2)):
        print(' '*(n-i-1), end=' ')
        print('*'*(2*i+1))
    
    for i in range(int((n-1)/2),-1,-1):
        print(' '*(n-i-1), end=' ')
        print('*'*(2*i+1))
    

File: Task-03/C/subtask3.c
#include<stdio.h>
int main() {
int n;
printf("Enter an integer:");
scanf("%d",&n);
if(n%2!=0){
for(int i=1;i<=n;i=i+2){
for(int j=i;j<=n;j=j+2)
printf(" ");
for(int k=1;k<=i;k++)
printf("*");
printf("\n");
}
for(int i=n-2;i>0;i=i-2){
for(int j=i;j<=n;j=j+2)
printf(" ");
for(int k=1;k<=i;k++)
printf("*");
printf("\n");
}
}
else{
for(int i=1;i<=n;i=i+2){
for(int j=i;j<=n;j=j+2)
printf(" ");
for(int k=1;k<=i;k++)
printf("*");
printf("\n");
}
for(int i=n;i>0;i=i-2){
for(int j=i;j<=n;j=j+2)
printf(" ");
for(int k=1;k<i;k++)
printf("*");
printf("\n");
}
}
return 0;
}
